### PR TITLE
Bug 1970975: upgrade test: expect upgrades to take longer on AWS

### PR DESCRIPTION
**Pull request by vrutkovs**
Due to AWS LB issue apiserver rollout takes twice as much. This bumps upgrade duration on AWS to 90 mins.

Upgrade tests:
* [test upgrade 4.7 https://github.com/openshift/origin/pull/26219 aws](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws/1403239794875568128)

```
[sig-cluster-lifecycle] cluster upgrade should complete in 60m (90m on AWS) | 1h7m50s
```

* [test upgrade 4.8.0-fc.8 https://github.com/openshift/origin/pull/26219 aws](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/release-openshift-origin-installer-launch-aws/1403239821593284608)
```
[sig-cluster-lifecycle] cluster upgrade should complete in 60m (90m on AWS)&nbsp;expand_less | 1h32m30s
-- | --
upgrade to registry.build01.ci.openshift.org/ci-ln-chzn6vb/release@sha256:1e6a8003e66a4b68e03128e204b679bd6db5ca354edb802f0709699b0490cde8 took too long: 92.50404250443333
```

### CODE REVIEW

File: test/e2e/upgrade/upgrade.go [hunk 1]
```diff
@@ -273,6 +273,13 @@ func clusterUpgrade(f *framework.Framework, c configv1client.Interface, dc dynam
 	// if upgrades take longer than this, then we will have a junit marker indicating failure.
 	durationToSoftFailure := 75 * time.Minute
 
+	infra, err := c.ConfigV1().Infrastructures().Get(context.Background(), "cluster", metav1.GetOptions{})
+	framework.ExpectNoError(err)
+	if infra.Status.PlatformStatus.Type == configv1.AWSPlatformType {
+		// due to AWS LB bug upgrades take longer on AWS
```
[the remark this answers is not in the document]
Comment (bparees) [reply test/e2e/upgrade/upgrade.go:279]: (in that their upgrades will take longer.  obviously they don't have the same timeout to worry about)

**Comment by bparees** [comment]
/lgtm

note, this might push us over the timeout limits on the multistep upgrade tests in the future (e.g. 4.8->4.9->4.10)

**Comment by vrutkovs** [comment]
>note, this might push us over the timeout limits on the multistep upgrade tests in the future (e.g. 4.8->4.9->4.10)

Right, we'll work with DPTP to extend those

File: test/e2e/upgrade/upgrade.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (smarterclayton) [reply test/e2e/upgrade/upgrade.go:279]: It's not the nu26 issue.  It's https://bugzilla.redhat.com/show_bug.cgi?id=1943804.  As a result, we take 12 extra minutes to rollout apiserver twice (120s -> 240s per machine, 3 machines, 2 separate rollouts, machines and apiserver)

[the remark this answers is not in the document]
Comment (smarterclayton) [reply test/e2e/upgrade/upgrade.go:279]: Go ahead and change the JUnit to be "Cluster upgrade should complete within 60m (90m on AWS)"

[the remark this answers is not in the document]
Comment (vrutkovs) [reply test/e2e/upgrade/upgrade.go]: Done. Added bug reference to the comment

**Comment by stbenjam** [comment]
@vrutkovs Do you have a new bug to use for this PR? Looks like the other one which you removed was the wrong bug.

/lgtm

**Pull request merged**